atrium-xrpc/src/lib.rs

```diff
@@ -29,7 +29,8 @@ where
 }
 
 /// An abstract HTTP client.
-#[async_trait]
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
```

Comment:
Ditto, it would be useful to mention this in the `atrium-xrpc` changelog (for the public traits that are changed).